src/envoy/http/service_control/filter_stats.h

```diff
@@ -29,7 +30,41 @@ namespace service_control {
 // clang-format off
 #define ALL_SERVICE_CONTROL_FILTER_STATS(COUNTER, HISTOGRAM)     \
   COUNTER(allowed)                                    \
-  COUNTER(denied) \
+  COUNTER(denied)                          \
+  COUNTER(check_count_0) \
```

Comment:
See if you can reduce code duplication. At the very least, you can use another macro to generate 0-16 for check vs report.

Also consider better naming instead of 0-16.

Reply:
For macro, 
- I thought it is not recommended in Google and for here it will be used once, so do we really need create a macro? What do you think? @qiwzhang 

For naming, I have double checked with 	
Tomaszand this is what they want.

Reply:
I prefer to name the counter properly,  it will be easier to read

Reply:
Done with use status name.

As for reuse macro, talking with Teju offline, seems not so trivial. Make it as a TODO.